src/cli/Add.cpp

```diff
@@ -62,10 +62,6 @@ Add::Add()
     positionalArguments.append({QString("entry"), QObject::tr("Path of the entry to add."), QString("")});
 }
 
-Add::~Add()
-{
-}
```

Comment:
I believe empty constructors were necessary in certain situations where Qt smart pointers were used. But if it compiles and CI doesn't complain, I'm okay with removing them.